### PR TITLE
settings.py: Add DIRs for templates and static files

### PR DESCRIPTION
**Pull request by sagar-kalra**
Add directories for organising templates and static files

Fixes https://github.com/pclubuiet/website/issues/9

### CODE REVIEW

File: pclubWebsite/settings.py [hunk 1]
```diff
@@ -120,7 +120,7 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR,'home','static')
```
Comment (gurpreetsingh00885) [inline pclubWebsite/settings.py]: Create a 'static' folder in the project root directory too and remove home from STATIC_ROOT too.

Comment (sagar-kalra) [reply pclubWebsite/settings.py]: Why to create a 'static' folder?

Comment (gurpreetsingh00885) [reply pclubWebsite/settings.py]: Its the directory where 
```python manage.py collectstatic```
will 'collect' static files for serving.

**Comment by sagar-kalra** [comment]
Is it okay now, @gurpreetsingh00885 ?

**Comment by gurpreetsingh00885** [comment]
I see you have made the change in the settings.py but haven't actually moved the static folder to the project root (where staticFiles and templates directories are).
Do this and squash your commits too.
Rebase is also required.

**Comment by sagar-kalra** [comment]
i added the folder, but why isnt it showing here. @gurpreetsingh00885

**Comment by gurpreetsingh00885** [comment]
It has not been added yet. Did you include it in your commit?

**Comment by gurpreetsingh00885** [comment]
And please rebase since we added travis CI. Checks won't pass until you do so.

**Comment by sagar-kalra** [comment]
is it done now?
@gurpreetsingh00885

**Comment by gurpreetsingh00885** [comment]
yes its done now :)

**Comment by sagar-kalra** [comment]
Thank God!! :+1:

**Pull request merged**